roles/repositories/README.md

```diff
@@ -0,0 +1,115 @@
+theforeman.foreman.repositories
+===============================
+
+This role defines Products and enabled Red Hat Repositories and Custom Repositories.
```

Comment:
```suggestion
This role defines Products and Custom Repositories and enables Red Hat Repositories.
```

Reply:
Yeah, this feels a lot more natural. :+1: